src/MusicStore/Controllers/HomeController.cs

```diff
@@ -28,12 +37,13 @@ public class HomeController : Controller
                 {
                     // Refresh it every 10 minutes.
                     // Let this be the last item to be removed by cache if cache GC kicks in.
-                    cache.Set(
+                    cache.SetExt(
                         cacheKey,
                         albums,
                         new MemoryCacheEntryOptions()
-                            .SetAbsoluteExpiration(TimeSpan.FromMinutes(10))
-                            .SetPriority(CacheItemPriority.High));
+                            .SetAbsoluteExpiration(TimeSpan.FromSeconds(_appSettings.CacheTimeoutSeconds > 0 ? _appSettings.CacheTimeoutSeconds : 1))
```

Comment:
This seems odd. We should not be calling the expiration timeotu if there is none.